docker-images/Dockerfile.python-connector-base

```
@@ -13,7 +13,7 @@ RUN adduser --uid 1000 --system --group --no-create-home airbyte && \
     chown airbyte:airbyte /airbyte
 
 ENV PIP_CACHE_DIR=/custom_cache/pip
-RUN pip install --upgrade pip==24.0 setuptools==70.0.0
+RUN pip install --upgrade pip==24.0 setuptools==78.1.1 
```

Nit: Trailing space.

```suggestion
RUN pip install --upgrade pip==24.0 setuptools==78.1.1
```

---

(The above can be auto-applied but it will kill existing test runs when it kicks off the new CI builds.)